rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -832,7 +854,7 @@ def _predict_label(
         if predict_out is None:
             return label, label_ranking
 
-        message_sim = predict_out["i_scores"].numpy()
+        message_sim = predict_out["i_scores"]
```

Comment:
For my understanding: What changed because of which we do not need to convert from `tf.Tensor` to numpy equivalent tensors?

Reply:
The outputs are no `tf.Tensor` anymore. Not sure where exactly the outputs are converted (tried to find it inside `fit`, but could not find it), but we already have numpy arrays here.

Reply:
Interesting 🤔